api/tests/Entity/TopBooksTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Entity;
+
+use ApiPlatform\Core\Bridge\Symfony\Bundle\Test\ApiTestCase;
+use App\DataProvider\TopBookCollectionDataProvider;
+use App\Entity\TopBook;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * These tests are read only, thus there is not need to use the RefreshDatabaseTrait.
+ */
+class TopBooksTest extends ApiTestCase
+{
+    /**
+     * @see TopBookCollectionDataProvider::getCollection()
+     */
+    public function testGetCollection(): void
+    {
+        // The client implements Symfony HttpClient's `HttpClientInterface`, and the response `ResponseInterface`
+        $response = static::createClient()->request('GET', '/top_books');
```

Comment:
`static::createClient()` can be set in `setup` PHPUnit function, which is called before each test function